src/app/modules/form/components/input-text/input-text.component.html

```diff
@@ -0,0 +1,6 @@
+<div class="wrapper">
+  <mat-form-field class="example-full-width" appearance="fill">
+    <mat-label>Recipe</mat-label>
```

Comment:
this label shouldn't be hardcoded here. You should use variables.
(same for placeholder)